src/EnergyPlus/DataSizing.hh

```diff
@@ -654,6 +654,10 @@ namespace DataSizing {
         Real64 applyTermUnitSizingHeatFlow(Real64 heatFlowWithOA, // Heating flow rate with MinOA limit applied
                                            Real64 heatFlowNoOA    // Heating flow rate without MinOA limit applied
         );
+
+        Real64 applyTermUnitSizingCoolLoad(Real64 coolFLoad); // Cooling load
+
+        Real64 applyTermUnitSizingHeatLoad(Real64 heatLoad); // Heating load
```

Comment:
There is a typo here in the argument name, but no need to commit just for that.  

I do have another note for later:  in absolutely tiny little functions like these, you _can_ put them in the struct declaration itself.  In this case, they are also const on the struct since they don't modify any struct member variables.  So it would look like:

```c++
struct Whatever {
  //...
  Real64 applyTermUnitSizingCoolLoad(Real64 const coolLoad) const {
    return coolLoad * this->SpecDesSensCoolingFrac;
  }
  Real64 applyTermUnitSizingHeatLoad(Real64 const coolLoad) const {
    return heatLoad * this->SpecDesSensCoolingFrac;
  }
  // etc.
}
```

and nothing in the cc file.  If the functions get more complicated, or do anything with a state variable, they should go in the cc, so there is definitely not anything you need to do here.  Just a note.